packages/core/content-type-builder/admin/src/components/FormModal/index.js

```diff
@@ -551,7 +562,7 @@ const FormModal = () => {
         // Add/edit a field to a created component (the end modal is not step 2)
       } else if (isCreatingCustomFieldAttribute) {
         const customFieldAttributeUpdate = {
-          attributeToSet: { ...modifiedData, customField: customFieldUid, type: customField.type },
+          attributeToSet: { ...modifiedData, customField: customFieldUid },
```

Comment:
Was the type not used anymore?

Reply:
It's now being set in the reducer instead of here

Reply:
Are we sure the type in always the same between the FormModal and the ListPage?

Reply:
Yes, when logging the `modifiedData` in the ListPage and in the FormModal the `type` is always the same between the two when creating or editing a custom field attribute in a content-type or component.